running-locally.adoc

```diff
@@ -1,64 +1,44 @@
+The *raincatcher-demo-cloud* app and *raincatcher-demo-auth* app requires a local running MongoDB instance. See the link:https://docs.mongodb.com/manual/installation/[MongoDB Installation Guide] for details on how to install MongoDB.
 
 [[redis]]
-Redis
-^^^^^
+=== Redis
 
-The *raincatcher-demo-cloud* app requires a running instance of Redis. See the link:https://redis.io/topics/quickstart[Redis Quick Start Guide] for details on how to insall Redis manually, or use your system's package manager.
+The *raincatcher-demo-cloud* app requires a local running instance of Redis. See the link:https://redis.io/topics/quickstart[Redis Quick Start Guide] for details on how to install Redis, or use your system's package manager.
 
-[[ruby-optional]]
-Ruby
-^^^^
+[[ruby]]
+=== Ruby 
 
-The link:https://github.com/feedhenry-raincatcher/raincatcher-cli[Raincatcher CLI Tool] is a useful tool for getting set up with all of the Raincatcher modules and Demo Apps. Follow the steps in the README file to get set up for local development.
+To use the Raincatcher CLI Tool, described later, you need to install link:https://www.ruby-lang.org/en/documentation/installation/[Ruby]. If the CLI tool is not required, you do not need to install Ruby.
 
-In order to use the link:https://github.com/feedhenry-raincatcher/raincatcher-cli[Raincatcher CLI Tool] to manage starting all of the demo apps, you will need to install link:https://www.ruby-lang.org/en/documentation/installation/[Ruby]
 
 [[install-raincatcher-cli-and-clone]]
-Install Raincatcher CLI & Clone Repositories
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+== Install Raincatcher CLI and Clone Repositories
+ The link:https://github.com/feedhenry-raincatcher/raincatcher-cli[Raincatcher CLI Tool] is a useful tool for getting set up with all of the Raincatcher modules and demo apps. Follow the steps in the README to clone, install, and link all of the Raincatcher modules.
 
-The *raincatcher-appforms* module requires that the containing cloud app has access to a running RHMAP MBaaS. This allows the local *raincatcher-demo-cloud* app to access The link:https://github.com/feedhenry-raincatcher/raincatcher-cli[Raincatcher CLI Tool] is a useful tool for getting set up with all of the Raincatcher modules and Demo Apps. Follow the steps in the README to clone, install, and link all of the Raincatcher modules.
-
-Configure Environment Variables
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+=== Configure Environment Variables
 
 In order to communicate with RHMAP and the local authentication service, several environment variables need to be configured. You can find the values in RHMAP -> Projects -> _Project name given in step 2 of link:getting-started.adoc[Getting Started Guide] -> WFM Demo Cloud App -> Environment Variables -> System Environment Variables.
```

Comment:
It looks like there's an underscore missing at the end of step 3 to make it italicized.